code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -262,8 +258,10 @@
 /datum/reagent/medicine/C2/syriniver/on_mob_life(mob/living/carbon/M)
 	M.adjustOrganLoss(ORGAN_SLOT_LIVER, 0.8)
 	M.adjustToxLoss(-1*REM, 0)
-	for(var/datum/reagent/toxin/R in M.reagents.reagent_list)
-		M.reagents.remove_reagent(R.type,1)
+	for(var/datum/reagent/R in M.reagents.reagent_list)
+		if(R == src)
+			continue
+		M.reagents.remove_reagent(R.type,0.4)
```

Comment:
won't this purge musiver?

Reply:
if so, congratulations, you've made each 5u injection of syriniver (assuming a fully-healed liver) heal 12 points of toxin damage less than it did before and have made the chem less intuitive to understand for new chemists

Reply:
43 points of toxin healing per 5u injection -> 31 points of toxin healing per 5u injection

Reply:
nightcoding :tm: